indigo_app/templates/place/tabbed_layout.html

```diff
@@ -35,6 +35,7 @@
       <a class="nav-item nav-link {% if view.tab == 'tasks' %}active{% endif %}" href="{% url 'tasks' place=place.place_code %}">
         Tasks <span class="badge badge-light">{{ place.tasks.unclosed.count }}</span>
       </a>
+      <a class="nav-item nav-link {% if view.tab == 'activities' %}active{% endif %}" href="{% url 'place_activities' place=place.place_code %}">Activities</a>
```

Comment:
I think the title "Activity" makes more sense.

Reply:
agreed